data/sql/updates/pending_db_world/rev_1598303195521812924.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598303195521812924');
+DELETE FROM `smart_scripts`WHERE `entryorguid`= 28202;
```

Comment:
this `DELETE` is bad! they primary key of `smart_scripts` is `entryorguid` AND `source_type`. If you don't specify the `source_type`, you can accidentally remove other scripts. Be careful when using `DELETE`